lib/serverless-tools/deployer/lambda_updater.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ServerlessTools
+  module Deployer
+    class LambdaUpdater
+      def initialize(pusher:, client:)
+        @client = client
+        @pusher = pusher
+      end
+
+      def update(config:)
+        remote_assets = pusher.push(config: config)
```

Comment:
Could there be an argument for passing the result of the `push` into this method, rather than calling it here (effectively making this a `push_and_update`)?